src/query/ast/src/parser/statement.rs

```diff
@@ -1506,7 +1488,7 @@ pub fn statement(i: Input) -> IResult<StatementMsg> {
             | #show_roles : "`SHOW ROLES`"
             | #create_role : "`CREATE ROLE [IF NOT EXISTS] '<role_name>']`"
             | #drop_role : "`DROP ROLE [IF EXISTS] '<role_name>'`"
-            | #create_udf : "`CREATE FUNCTION [IF NOT EXISTS] <udf_name> (<parameter>, ...) -> <definition expr> [DESC = <description>]`"
+            | #create_udf : "`CREATE FUNCTION [IF NOT EXISTS] <udf_name> AS {(<parameter>, ...) -> <definition expr> | (<arg_type>, ...) -> <return_type> ADDRESS <udf_server_address>} [DESC = <description>]`"
```

Comment:
How about using `ADDRESS=<udf_server_address>` to align with other options?

Reply:
Seems good.